www/js/index.js

```diff
@@ -101,6 +105,13 @@ var boot = function ()
 		});
 	}, false);
 
+	clear_button.addEventListener('click', function (evt)
+	{
+		exec_button.disabled = true;
```

Comment:
You'll want to remove this line.  Otherwise, the exec_button will remain disabled after clearing the input/output.